install/updates/20-indices.update

```diff
@@ -267,3 +267,43 @@ default: objectClass: nsIndex
 only: nsSystemIndex: false
 only: nsIndexType: eq
 only: nsIndexType: sub
+
+dn: cn=description,cn=index,cn=userroot,cn=ldbm database,cn=plugins,cn=config
+default: cn: description
+default: objectclass: top
+default: objectclass: nsindex
+only: nssystemindex: false
```

Comment:
I see that only is used in a lot of other places in this update file but I think default is more appropriate.

Reply:
`default` won't update the entry as we want it should the entry with such an attribute exist. A good example is the `fqdn` indices entry - should we add `default:nsIndexType: sub`, the `sub` operation type does not get added, we have to use `only` for all the index types.

Reply:
For _new_ indexes then "default" should be fine. For updating _existing_ indexes then "only" is the way we've managed it in the past ("add" should work as well). "only" is a bit of a hammer since it would eliminate any end-user changes to the index.

Reply:
This is quite ambivalent, there would be reason to keep `only` so that any existing configuration we did not create can get overwritten in the same manner as our managed configuration. By putting `default` there, we still consider any possible existing configuration inadequate but are deciding not do do anything with it, which does not destroy the user config yet won't work in the way we want 🤕 

Please, help me decide here. Do we really want to go with `default`?

Reply:
IPA tries to make good default decisions for users. If this is anything other than default then on every upgrade the index is doing to revert back. This could be a bad experience. We've done this in places where we had the original index types incorrect to force the matter but as general principal I'd prefer to leave it up to the user to decide if they want additional or no types.

This will affect their installation but it is theirs after all. I'd go with default.

Reply:
That's a good reasoning, updating. hopefully I did not forget anything.